### ISSUE TITLE
ImportError: DLL load failed while importing _c_internal_utils: The specified module could not be found.

### ISSUE THREAD

**Issue by danielgatis** [pyinstaller/pyinstaller#8713]
Hello 👋 , 

after I updated the version of Matplotlib, I'm getting this error. Any ideas?

```
RuntimeError: Child process call to mpl_data_dir() failed with:
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\PyInstaller\isolated\_child.py", line 63, in run_next_command
    output = function(*args, **kwargs)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\PyInstaller\hooks\hook-matplotlib.py", line 19, in mpl_data_dir
    import matplotlib
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\matplotlib\__init__.py", line 159, in <module>
    from . import _api, _version, cbook, _docstring, rcsetup
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\matplotlib\cbook.py", line 32, in <module>
    from matplotlib import _api, _c_internal_utils
ImportError: DLL load failed while importing _c_internal_utils: The specified module could not be found.
Error: Process completed with exit code 1.
```

Here is my spec file: https://github.com/danielgatis/rembg/blob/main/rembg.spec
here is the exe build script: https://github.com/danielgatis/rembg/blob/main/_build-exe.ps1

Thank you so much

**Comment by bwoodsend** [comment]
> Any ideas?

Unfortunately I do. See https://github.com/matplotlib/matplotlib/issues/28551#issuecomment-2267563049

I suggest you downgrade to matplotlib 3.9.0 whilst matplotlib figure their wheels out.

**Comment by tacaswell** [comment]
The 3.9.1 wheels on windows would, depending on import order, cause segfaults, see https://github.com/matplotlib/matplotlib/issues/28551#issuecomment-2266699619.

We deleted the bad wheels to avoid segfaults (which seemed like a good idea) but failed to anticipate that this would cause downstream projects to start trying to build mpl from source instead of pulling the 3.9.0 wheel.  We have now yanked the whole 3.9.1 release to fix these problems and will get 3.9.2 out as quick as we can.

To prevent this sort of thing in the future (trying to build something from source when you are not prepared to do so) I suggest using `matplotlib --only-binary "matplotlib" ` in your requirements file (which may be a good practice for all wheels that have extensions).

Sorry for the trouble.
